### PR TITLE
rustdesk: 1.1.8 -> 1.1.9, fix

### PR DESCRIPTION
**Pull request by marius851000**
###### Description of changes

Changelog : https://github.com/rustdesk/rustdesk/releases/tag/1.1.9

###### Things done

Updated to version 1.1.9, and fixed compilation (which failed due to removed remote file or something similar). I did the minimum to make Vcpkg work, but I might take a better look later to better see how to integrate it with Nix (if it hasn't already -- I have no idea what vcpkg is, and I got the information of the format by looking at the code of the Rust vcpkg crates)

Tested a simple session between my computer and my phone, it work.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by marius851000** [comment]
Forgot to ping the package manager, @Leixb

**Comment by Leixb** [comment]
Result of `nixpkgs-review pr 174966` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>rustdesk</li>
  </ul>
</details>

File: pkgs/applications/networking/remote/rustdesk/default.nix [hunk 1]
```diff
@@ -51,10 +52,29 @@ rustPlatform.buildRustPackage rec {
...
   in ''
     export VCPKG_ROOT="$TMP/vcpkg";
 
+    mkdir -p $VCPKG_ROOT/.vcpkg-root
     mkdir -p $VCPKG_ROOT/installed/${vcpkg_target}/lib
+    mkdir -p $VCPKG_ROOT/installed/vcpkg/updates
+    ln -s ${updates_vcpkg_file} $VCPKG_ROOT/installed/vcpkg/status
+    mkdir -p $VCPKG_ROOT/installed/vcpkg/info
+    touch $VCPKG_ROOT/installed/vcpkg/info/libyuv_1.0_x64-linux.list
+    touch $VCPKG_ROOT/installed/vcpkg/info/libvpx_1.0_x64-linux.list
```
Comment (Leixb) [inline pkgs/applications/networking/remote/rustdesk/default.nix:77]:
```suggestion
    touch $VCPKG_ROOT/installed/vcpkg/info/libyuv_1.0_${vcpkg_target}.list
    touch $VCPKG_ROOT/installed/vcpkg/info/libvpx_1.0_${vcpkg_target}.list
```

**Comment by happysalada** [comment]
@marius851000 happy to merge once you have taken care of the comment.

**Comment by marius851000** [comment]
I included the change (thank to ``git rebase HEAD~2 -i`` and ``git push --force``). I haven't tested yet.

**Comment by devhell** [comment]
@Leixb does this look good to you?

**Comment by Leixb** [comment]
Yeah LGTM. I have tested it and all seems to be working properly.

It's quite hacky, but it's the best we can do until we figure out a good way to add vcpkg into nixpkgs

**Pull request merged**